elements/bulbs-video/elements/meta.js

```diff
@@ -0,0 +1,66 @@
+import React, { PropTypes } from 'react';
```

Comment:
@daytonn From a few weeks later I'm not sure how vital it is to separate the outermost class from the 'Root' class. Probably not.